docs/development.md

```diff
@@ -18,6 +18,28 @@
 
 # Developer's guide
 
+To run Vagrant development environment, the `$SWAN_DEVELOPEMENT_ENVIRONMENT` variable must be set.
```

Comment:
@skonefal - don't you think that this information belongs to [vagrant readme](https://github.com/intelsdi-x/swan/blob/master/vagrant/README.md)?